src/i18n/en.json

```
@@ -55,8 +55,8 @@
     "datahub_not_active_title":"Track user behavior and optimize your Marketing actions",
     "datahub_not_domains_title":"Track user behavior and optimize your Marketing actions",
     "no_domains_HTML": "<p>How do users behave on your Website or E-commerce? Track their behavior for a period of time and find out which are the most visited pages, how many of those visits have an Email identified by Doppler and how many don’t. By tracking the user’s journey you'll be able to detect vanishing points and opportunities for improvement!</p><p>You haven’t added a domain yet. You can do so in the <a href=\"{dopplerBaseUrl}/ControlPanel/CampaignsPreferences/SiteTrackingSettings\">On-Site Tracking</a> option in the Control Panel.</p>",
-    "upgrade_account_free_HTML": "<p>How do users behave on your Website or E-commerce? Track their behavior for a period of time and find out which are the most visited pages, how many of those visits have an Email identified by Doppler and how many don’t. By tracking the user’s journey you'll be able to detect vanishing points and opportunities for improvement!</p><p>This functionality is enabled only for accounts with <a href=\"https://app2.fromdoppler.com/ControlPanel/AccountPreferences/UpgradeAccount?Plan=monthly\">Paid Plan</a></p>",
-    "upgrade_account_free_title": "Track user behavior and optimize your Marketing actions"
+    "upgrade_account_free_HTML": "<p>Upgrade your account and access detailed Reports on the behavior of users on your Website or E-commerce. Discover which are the most visited pages, how many visitors have an Email that Doppler has identified and how many don't.</p><p>By joining any Paid Plan you can enjoy FOR FREE this feature. Limited time only. <a href=\"https://app2.fromdoppler.com/ControlPanel/AccountPreferences/UpgradeAccount?Plan=monthly\">UPGRADE NOW</a></p><p>Any doubts? Press <a target=\"_blank\" href=\"https://help.fromdoppler.com/en/advanced-reports\">HELP.</a></p>",
```

> `[...] <a href=\"https://app2.fromdoppler.com/ControlPanel/AccountPreferences/UpgradeAccount?Plan=monthly\"> [...]`

⚠️ The link cannot be hardcoded, we need to base the link in current environment base URL

---

Something like: 

```suggestion
    "upgrade_account_free_HTML": "<p>Upgrade your account and access detailed Reports on the behavior of users on your Website or E-commerce. Discover which are the most visited pages, how many visitors have an Email that Doppler has identified and how many don't.</p><p>By joining any Paid Plan you can enjoy FOR FREE this feature. Limited time only. <a href=\"{baseUrl}/ControlPanel/AccountPreferences/UpgradeAccount?Plan=monthly\">UPGRADE NOW</a></p><p>Any doubts? Press <a target=\"_blank\" href=\"https://help.fromdoppler.com/en/advanced-reports\">HELP.</a></p>",
```

And we should pass the baseUrl where the literal is being used.